portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,7 +34,7 @@
 import java.util.List;
 import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments **/
```

Comment:
As a heads up, the `JavaDoc` comment format (and I think some other languages as well) is that you use two `*`s at the beginning and then one at the end. A lot of IDEs will distinguish that comments that start with `/**` instead of `/*` are docs. There's nothing wrong with `**/` at the end, just wanted to give you a heads up :slightly_smiling_face:

Reply:
I didn't know about using `/**` to distinguish the start of a comment, I just used two stars to make both ends look the same. Thanks for the heads up.